mkdocs.yml

```diff
@@ -57,15 +57,21 @@ nav:
   - 'Overview': index.md
   - 'Documentation': overview.md
   - 'Concepts': concepts.md
-  - 'Kotlin':
-    - 'Overview': kotlin/index.md
-    - 'API Reference':
+  - 'API Reference':
+    - 'Kotlin':
       - 'workflow-core': 'kotlin/api/workflow-core/com.squareup.workflow/index.md'
       - 'workflow-runtime': 'kotlin/api/workflow-runtime/com.squareup.workflow/index.md'
       - 'workflow-ui-core': 'kotlin/api/workflow-ui-core/com.squareup.workflow.ui/index.md'
       - 'workflow-ui-android': 'kotlin/api/workflow-ui-android/com.squareup.workflow.ui/index.md'
       - 'workflow-testing': 'kotlin/api/workflow-testing/com.squareup.workflow.testing/index.md'
       - 'workflow-rx2': 'kotlin/api/workflow-rx2/com.squareup.workflow.rx2/index.md'
+    - 'Swift':
+      # mkdocs will warn about these not existing, that's fine, deploy_website.sh manually adds them.
```

Comment:
Perhaps echo this reassurance before the warning shows up.

Reply:
I can't echo it from here, and I don't want to duplicate documentation everywhere. If you try fixing it, you'll see this comment.